PlaygroundApp/ContentView.swift

```diff
@@ -1,9 +1,85 @@
 import SwiftUI
 
 struct ContentView: View {
+
+    var body: some View {
+        VStack(spacing: 20) {
+            StatefulTextWithButton()
+            Matrix()
+            DummyList()
+        }
```

Comment:
I've split this in dedicated subviews to hopefully make it easier to follow. 

As we add more utilities, I can see this becoming a list where each item loads a different screen for a different group of tests.